routes/botSettings.js

```diff
@@ -0,0 +1,34 @@
+const express = require('express')
+const botSettings = require('../db/botSettings.js')
+const router = express.Router()
+
+module.exports = router;
+// reset endpoint, use only in testing
+router.get('/start', async (req, res) => {
+    await botSettings.destroy({where: {}})
+    let bot = await botSettings.build({'id': 1,
+                    'botToken': 'Token', 
```

Comment:
Formatting is particularly odd here, fix it. Try using automatic formatting to achieve that. https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode

Also read about linters and add one to the project. Add the one that is most popular for express. https://eslint.org

Reply:
This file renamed to botSetting.js check it plz